Dockerfile

```diff
@@ -61,6 +61,17 @@ RUN wget https://github.com/Metaswitch/calico-docker/releases/download/v0.5.1/ca
     chmod +x calicoctl && \
     mv calicoctl /usr/local/bin/
 
+#######################
+# Star (test workload)
+#######################
+WORKDIR /star
+
+RUN wget http://downloads.mesosphere.io/demo/star/v0.5.0/star-collect-v0.5.0-linux-x86_64
+RUN wget http://downloads.mesosphere.io/demo/star/v0.5.0/star-probe-v0.5.0-linux-x86_64
```

Comment:
You can use

`ADD http://downloads.mesosphere.io/demo/star/v0.5.0/star-collect-v0.5.0-linux-x86_64 /star/`

Also, probably worth `chmod`ing the files here, in case someone forgets in their app JSON.